UT4MasterServer/Controllers/ErrorsController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using UT4MasterServer.Exceptions;
+
+namespace UT4MasterServer.Controllers;
+
+[ApiController, ApiExplorerSettings(IgnoreApi = true)]
+[Route("api/errors")]
+public class ErrorsController : Controller
```

Comment:
Does this have to inherit `Controller` or would `ControllerBase` suffice?

Reply:
No, it is not necessary to inherit Controller. I will change it.